src/test/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/reporting/MoveReportFiltererTest.kt

```diff
@@ -115,10 +130,17 @@ internal class MoveReportFiltererTest{
         assertThat(longHaulReports).isEqualTo(setOf(completedLongHaul))
     }
 
-//    @Test
-//    fun `Unbillable and multi-type moves`() {
-//
-//        val multiTypeReports = MoveReportFilterer.longHaulReports(filter, reports).toSet()
-//        assertThat(lodgingReports).isEqualTo(setOf(completedLodging))
-//    }
+    @Test
+    fun `Only lockout moves within date range are filtered`() {
+
+        val lockoutReports = MoveReportFilterer.lockoutReports(filter, reports).toSet()
+        assertThat(lockoutReports).isEqualTo(setOf(completedLockout))
+    }
+
+    @Test
+    fun `Unbillable and multi-type moves`() {
+
+        val multiTypeReports = MoveReportFilterer.multiTypeReports(filter, reports).toSet()
+        assertThat(multiTypeReports).isEqualTo(setOf(multiTypeMove, completedUnbillable))
```

Comment:
On this one could do - `assertThat(multiTypeReports).containsExactlyInAnyOrder(multiTypeMove, completedUnbillable)`